### PR TITLE
Van stylekit first demo

### PR DESCRIPTION
**Pull request by vanaudiobuddy**
This pull request includes:
- an extension to UIView, which contains a string property via associative reference for tagging.
- a file that contains property names of some UI elements.

### CODE REVIEW

File: SwiftStyleKitSample/StyleKitSample/Style.swift [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+            imageNames = json["Images"] as? [String: String]
+            
+            if let labelDict = json["Labels"] as? [String: [String:AnyObject]] {
+                for (labelKey, specification) in labelDict {
+                    labelStyles[labelKey] = serializeLabelSpec(specification)
+                }
+            }
+            
+            if let buttonDict = json["Buttons"] as? [String: [String:AnyObject]] {
+                for (labelKey, specification) in buttonDict {
```
Comment (joxenford) [inline SwiftStyleKitSample/StyleKitSample/Style.swift]: buttonDict has a labelKey?  Should this name be clearer?

File: SwiftStyleKitSample/StyleKitSample/Style.swift [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+                    buttonStyles[labelKey] = serializeButtonSpec(specification)
+                }
+            }
+            
+            if let itemDict = json["TextFields"] as? [String: [String:AnyObject]] {
+                for (labelKey, specification) in itemDict {
```
Comment (joxenford) [inline SwiftStyleKitSample/StyleKitSample/Style.swift]: itemDict has a labelKey?  Again, should he name be clearer?

File: SwiftStyleKitSample/StyleKitSample/Style.swift (hunk 1 above)
Comment (vanaudiobuddy) [inline SwiftStyleKitSample/StyleKitSample/Style.swift]: You're right. I was copying from the previous block so I was not aware of the naming. I'll fix it.

Comment (vanaudiobuddy) [inline SwiftStyleKitSample/StyleKitSample/Style.swift]: Actually, that was Eric's code. Should I change it?

**Pull request merged**